app/src/androidTest/java/com/github/h3lp3rs/h3lp/GuideTest.kt

```diff
@@ -19,7 +19,7 @@ import org.junit.Test
 import org.junit.runner.RunWith
 
 @RunWith(AndroidJUnit4::class)
-class GuideTest {
+class GuideTest : H3lpAppTest() {
 
     private fun launch(): ActivityScenario<MainPageActivity> {
         return launch(Intent(getApplicationContext(), MainPageActivity::class.java))
```

Comment:
True, but it takes as argument an intent that depends on the activity. On the other hand, it would be easier to understand what it does if we abstract it ;)